streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -616,6 +619,12 @@ public boolean process(final long wallClockTime) {
         return true;
     }
 
+    @Override
+    public void recordTotalLatency(final long elapsedLatencyMs) {
```

Comment:
prop: Could you rename `recordTotalLatency()` to `recordProcessRatioAndReset()`.